### ISSUE TITLE
Validation data index is strange

### ISSUE THREAD

**Issue by ghost** [dhruvramani/C2AE-Multilabel-Classification#7]
File "model/src/dataset.py", line 32,
validation data X, Y is overlapped with train data X, Y.
I think it should be modified as follow.
X, Y = X[0 : int(0.2 * length) , :], Y[0 : int(0.2 * length), :]
=> X, Y = X[int(0.8 * length) : length, :], Y[int(0.8 * length) : length, :]

**Comment by greeness** [comment]
I agree that the validation data should not overlap with training data.
